interface/patient_file/transaction/transactions.php

```diff
@@ -13,15 +13,11 @@
 ?>
 <html>
 <head>
-<?php html_header_show();?>
```

Comment:
I wouldn't remove the above, since it seems that some folks use this as a hook for custom stuff.